custom/icds_reports/tests/fixtures/dashboard_daily_feeding.csv

```diff
@@ -1,212 +1,212 @@
 doc_id,inserted_at,repeat_iteration,timeend,state_id,received_on,child_health_case_id,eligible,active,attended_child_ids,breakfast,lunch,double_meal,supervisor_id
```

Comment:
see the query in commit message used as logic to map the doc_id  from daily feeding fixture to dashboard_daily_feeding.csv

Reply:
I have changes this to include the query change in the test cases. Otherwise the query change was not showing any change to the test cases.